### DISCUSSION TITLE
Unable to save & load HTML & Styles from db in latest grapesjs@0.20.1

### DISCUSSION THREAD

**Discussion by Mayank-wings** [GrapesJS/grapesjs#4725] [Q&A]
**I init grapesjs and all the component ,style manager, blocks , tailwindblocks all the things working fine  with the Next.js** 

![Screenshot from 2022-11-11 00-04-13](https://user-images.githubusercontent.com/115448445/201268254-1b0ce414-d5a3-42f6-92c1-d52c905529ac.png)
![Screenshot from 2022-11-11 00-31-06](https://user-images.githubusercontent.com/115448445/201270274-c7958788-df9f-48a0-abb1-7435a51e69f0.png)

I have pass the `urlLoad `and `urlStore` like this 

when i am using the latest Grapesjs  `0.20.1 ` version i can not save my components, styles , html to database but when i switch to Grapesjs `0.17.22` i can save the html, component, style to database and also can load the data from database. 

I solve this issue from the discussion https://github.com/artf/grapesjs/discussions/4571 where i came to know about that the official doc is not updated  and all the updated details available at [Release Notes](https://github.com/artf/grapesjs/releases)

can some one tell me how to save & load the Html, Component , Styles, Assets from db with `0.20.1`?

**Comment by vizardkill** [comment]
In the Grapes versions, the data will be saved from keys like 'gjs-pages', 'gjs-styles', 'gjs-assets', that changed in the last version to, styles, assets and pages

**Comment by artf** [comment]
@Mayank-wings docs are already updated and [here](https://grapesjs.com/docs/modules/Storage.html#html-code-with-project-data) you can see how you can enrich your project data with stuff like HTML

**Reply by clonefunnels** [reply]
I only want to load ONE page at a time like I always have before. Can you please just give me a simpler way to load ONE CSS call and ONE HTML call, and be done with it?  Without having every client install their own Rest API server.  That is going to be hell for everyone.
